public_html/wp-content/mu-plugins/redirect.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace WordCamp\Redirect;
+
+defined( 'WPINC' ) || die();
+
+add_action( 'init',  __NAMESPACE__ . '\redirect_subdomain' );
+
+/**
+ * Redirects the root URL of the "events.wordcamp.test" to "https://make.wordpress.org/community/events".
+ */
+function redirect_subdomain() {
+	if ( 'events.wordcamp.test' === $_SERVER['HTTP_HOST'] && wp_using_themes() && ! defined( 'REST_REQUEST' ) ) {
```

Comment:
I imagine we'll want to have a separate folder for the Events network (see #906), so IMO it'd be good to make this `mu-plugins/events/redirects.php` (plural since we'll probably add more in the future).

Since it's in a subfolder, it wouldn't be included automatically. You could include the file from `load-other-mu-plugins.php`, but only if it's on the events network. Then that condition wouldn't be needed in this file.